app/src/main/java/com/islaharper/dawnofandroid/presentation/common/FlavourItem.kt

```diff
@@ -0,0 +1,139 @@
+package com.islaharper.dawnofandroid.presentation.common
+
+import android.content.res.Configuration
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.islaharper.dawnofandroid.R
+import com.islaharper.dawnofandroid.domain.model.Flavour
+import com.islaharper.dawnofandroid.presentation.components.RatingBar
+import com.islaharper.dawnofandroid.ui.theme.DawnOfAndroidTheme
+import com.islaharper.dawnofandroid.ui.theme.PADDING_LARGE
+import com.islaharper.dawnofandroid.ui.theme.PADDING_MEDIUM
+import com.islaharper.dawnofandroid.ui.theme.PADDING_SMALL
+import com.islaharper.dawnofandroid.util.Constants
+
+@Composable
+fun FlavourItem(
+    flavour: Flavour,
```

Comment:
Will these `FlavourItem`s get used in a LazyList? The reason I am asking is because one of the parameters is typed as `Flavour`. When I navigate to `Flavour`'s definition, there is a property declared as `var` so it is unstable. Maybe you can solve this problem in a future PR by passing in an immutable `ViewState.Flavour` or something else that is stable? The ViewModel could map the database's Flavour into the ViewState.